src/music/player.py

```diff
@@ -314,9 +316,10 @@ def on_read(ms, non_music):
 
                             @audio.event
                             def on_resolve(info, path):
-                                if(self.playlist.current().info == info): # TODO: fix if client skips song/video before finished downloading, current() will be None
-                                    self.add_to_footer(source= 'Youtube 📁', icon_url=YOUTUBE_ICON)
-                                    asyncio.run_coroutine_threadsafe(self.messenger.gui['player'].update(), self.connected_client.loop)
+                                if self.playlist.current():
+                                    if(self.playlist.current().info == info):
```

Comment:
Remove the  parenthesis around this if

Reply:
> Remove the parenthesis around this if
I would say to stay consistent two to either use parenthesis for all if's or not

Reply:
```suggestion
                                    if self.playlist.current().info == info:
```
I agree. No parenthesis for just a single mundane check.